less/core/scrolling.less

```diff
@@ -1,27 +1,22 @@
-.restrain-height() {
-  height: 100%;
-  overflow: hidden;
-}
-
 html.adapt-scrolling {
-  .restrain-height;
-
-  body, #app {
-    .restrain-height;
-  }
+  height: 100%;
+  // Allow overflow scroll top for refresh on android
+  overflow: auto;
 
+  // Offset the navigation bar width the the body scrollbar
   .nav {
-    position: relative;
+    // IE11 is consistant at 18px
+    width: calc(100% - 18px);
```

Comment:
should 18px not be converted into rem?

Reply:
No, the browser scrollbar in IE11 doesn't change with the body font size.